tests/parsers/google_logging.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for Google log parser"""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.lib import errors
+from plaso.parsers import google_logging
+
+from tests.parsers import test_lib
+
+
+class GooglelogTest(test_lib.ParserTestCase):
```

Comment:
Let's keep the name of this class consistent with most other parsers and rename to:  GoogleLogParserTest

Reply:
Done